guides/common/modules/proc_migrating-to-a-new-server.adoc

```diff
@@ -0,0 +1,47 @@
+[id="migrating-to-a-new-server_{context}"]
+
+= Migrating to a new server
+
+.Prerequisites
+* Install a minimal {RHEL} 8 instance on the target machine with the capacity to store backup files.
+* Do not install any {RHEL} 8 software groups or third-party applications.
+
+For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/index[Performing a standard {RHEL} installation].
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} to migrate to {RHEL} 8
+
+The fresh installation method has three steps:
+
+. Create a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 system source machine.
+. Perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on a minimal {RHEL} 8 instance on the target machine.
+. Restore the backup on the {RHEL} 8 instance.
+
+== Performing a fresh installation of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you have created a backup of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 7 source machine, you can install {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 target machine.
+
+* To install {ProjectServer}, see {AdministeringDocURL}: Administering_Project/{Accessing-Server}.
+* To install {SmartProxyServer}, see {AdministeringDocURL}: Administering_Project/{installing-capsule-server}.
+
+=== Creating a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 7
+
+Before you perform a fresh installation of the {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 system, back up your {ProjectServer} or {SmartProxyServer} data on the {RHEL} 7 system by creating an offline backup.
+
+If you recently created an offline backup, you can perform an incremental backup to update the existing backup.
+
+.Procedure
+* Perform an offline or incremental backup on the source machine:
+** To perform an offline backup, see {AdministeringDocURL}: Administering_Project/{Backing_Up_Server_and_Proxy}.
+** To perform an incremental backup, see {AdministeringDocURL}: Administering_Project/{Performing-an-Incremental-Backup}.
+* Transfer the backup to the target machine.
+
+=== Restoring a backup of {ProjectServer} or {SmartProxyServer} on {RHEL} 8
+
+After you perform a fresh installation of {ProjectServer} or {SmartProxyServer} on the {RHEL} 8 instance of the target machine, you can restore the backup you previously created.
+
+.Procedure
+
+* Update your network configuration to match the target server’s IP address and new host name.
```

Comment:
The IP address doesn't *need* to match, and the hostname is listed as a pre-requisite in https://docs.theforeman.org/nightly/Administering_Red_Hat_Satellite/index-foreman-el.html#Restoring_Server_or_Smart_Proxy_from_a_Backup_admin, so I wonder if this line adds any real value (but I am no docs expert, and maybe sometimes repeating important things is a good thing?)

Reply:
I can't find where it was, but somewhere along the way in the drafting of this text, an SME reviewer said this had to be the first step of this process or the entire thing wouldn't work.